examples/gross_pitaevskii.jl

```diff
@@ -5,6 +5,7 @@ using PyCall
 using DFTK
 using Printf
 using LinearAlgebra
+using PyPlot
```

Comment:
Do we want to go full Plots and drop PyPlot everywhere?

Reply:
Yes, I'm on that. There are a few tiny things though, which came up related to labels taking real parts, etc.

Reply:
It's really annoying that the default GR backend does not support unicode, actually.